Mathlib/RingTheory/Idempotents.lean

```diff
@@ -335,6 +345,143 @@ theorem CompleteOrthogonalIdempotents.of_ker_isNilpotent_of_isMulCentral
     simpa [RingHom.mem_ker, sub_eq_zero] using congr_fun h₂.symm i
   exact h₁
 
+omit [Fintype I] in
+lemma ne_sum_ortho {e : R} (he : IsPrimitiveIdempotents e)
```

Comment:
Looking at name `OrthogonalIdempotents.mul_eq`, maybe this one should be called `OrthogonalIdempotents.mul_eq.ne_add`

Reply:
Or maybe `IsPrimitiveIdempotents.xxx`

Reply:
deleted